src/lib/hooks/useContractFilter.ts

```diff
@@ -0,0 +1,17 @@
+import { useCallback } from "react";
+
+import type { ContractInfo } from "lib/types";
+
+export const useContractSearchFilter = (keyword: string) =>
```

Comment:
I don't think this need to be a hook at all. We can delete this file move the logic to the same file as useRecentContractsData.